infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -286,14 +286,16 @@ private void prefixTrim(CorfuPayloadMsg<TrimRequest> msg, ChannelHandlerContext
     @ServerHandler(type = CorfuMsgType.READ_REQUEST)
     private void read(CorfuPayloadMsg<ReadRequest> msg, ChannelHandlerContext ctx, IServerRouter r) {
         long address = msg.getPayload().getAddress();
-        log.trace("read: {}", msg.getPayload().getAddress());
+        boolean cacheable = msg.getPayload().isCacheableOnServer();
```

Comment:
Why not use `CacheOption` ?

Reply:
Actually my first version was using `CacehOption`, but I found this server option needs to be passed to client in the the message (in AddressSpaceView), and this creates confusion on client side.